draft-ietf-rtcweb-jsep.xml

```diff
@@ -1165,7 +1165,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                 </section>
 
                 <section title="createDataChannel" anchor="sec.createDataChannel">
-                  <t>[TODO]</t>
+                  <t>
+                    The createDataChannel method creates a data channel and
+                    attaches it to the PeerConnection. If no data channel exists
+                    on this PeerConnection, then creating a data channel will
+                    also require a new offer/answer exchange. All data channels on
+                    a given PeerConnection share the same SCTP/DTLS association
+                    and therefore the same m= section, so subsequent data channels
```

Comment:
'subsequent creation of data channels does not have any impact on the JSEP state'